neo/Network/P2P/Payloads/CosignerUsage.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Neo.IO;
+using Neo.IO.Json;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class CosignerUsage : ISerializable
+    {
+        public WitnessScope Scope;
+        public byte[] ScopeData;
```

Comment:
Should be UInt160

Reply:
But group data is also UInt160? I thought groups were organized as pubkeys, so size was supposed to be 33bytes in that case, that's why I haven't unified yet. Groups have some Uint160 id too? I didn't create tests for it yet.

Reply:
Please see: https://github.com/neo-project/neo/pull/968/commits/862ff4b7b1751b5a3a54697008a225d4422ca3e9#diff-3425c2ccbc7797fa6cdc01a9a753565bR178-R184

pubkey size is different than UInt160.